lld/test/ELF/linkerscript/overlay-symbol-assign.test

```diff
@@ -0,0 +1,32 @@
+# REQUIRES: x86
+# RUN: echo 'nop; .section .small, "a"; .long 0; .section .big1, "a"; .quad 1; .section .big2, "a"; .quad 2;' \
```

Comment:
Use `RUN: rm -rf %t && split-file %s %t && cd %t` pattern to prepare the input assembly file.
You can use `#--- a.s` for assembly and `#--- 1.lds` for the linker script.